manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "vendor": "vtex",
   "name": "search-graphql",
-  "version": "0.16.2",
+  "version": "0.17.1-hkignore.4",
```

Comment:
I am not sure, but shouldn't this be changed?

Reply:
Sure